packages/notification-service/test/transfers.test.ts

```diff
@@ -0,0 +1,144 @@
+import { Log, Response, Transfer } from '../src/blockscout/blockscout'
+import {
+  filterAndJoinTransfers,
+  handleTransferNotifications,
+  notifyForNewTransfers,
+} from '../src/blockscout/transfers'
+
+export let TRANSFER1: Transfer
+export let TRANSFER2: Transfer
+export let sendPaymentNotificationMock: any
+export let getLastBlockNotifiedMock: any
+export let setLastBlockNotifiedMock: any
+export let decodeLogsMock: any
+
+const response: Response<Log> = {
+  status: '',
+  result: [
+    {
+      transactionIndex: '',
+      transactionHash: '',
+      topics: [''],
+      timeStamp: '',
+      logIndex: '',
+      gasUsed: '',
+      gasPrice: '',
+      data: '',
+      blockNumber: '',
+      address: '',
+    },
+  ],
+  message: '',
+}
+
+jest.mock('../src/config', () => ({
+  BLOCKSCOUT_API: '',
+  STABLE_TOKEN_ADDRESS: '',
+  GOLD_TOKEN_ADDRESS: '',
+}))
+
+jest.mock('node-fetch', () => ({
+  ...jest.requireActual('node-fetch'),
```

Comment:
Is this line necessary?

Reply:
It's needed for the last test in testing 'should update the last set block number'